packages/application-shell/src/components/fetch-project/fetch-project.graphql

```diff
@@ -27,6 +27,15 @@ query ProjectQuery($projectKey: String!) {
       name
       value
     }
+    allAppliedDataFences {
+      ... on StoreDataFence {
+        type
+        storeKey
+        group
+        resourceAccessPermission
```

Comment:
I realise now that this is introducing a new term in the application-kit.
We always used `name` for it. I'll update the response accordingly